agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -431,20 +518,37 @@ function remove_agent {
 
       echo "* Deleting the sysdig-image-analyzer configmap"
       kubectl delete configmap sysdig-image-analyzer --namespace=$NAMESPACE
+    elif [ "$(kubectl get pods -n $NAMESPACE | grep -c "node-analyzer")" -ge 1 ]; then
```

Comment:
I'm not sure nesting quotes works this way. How about simply:

```suggestion
    elif [ "$(kubectl get pods -n $NAMESPACE | grep -c node-analyzer)" -ge 1 ]; then
```

Reply:
In this case it works because the inner quotes are inside the command substitution `$(...)`, but I do also prefer your suggestion